test/service/test_response.py

```diff
@@ -1442,7 +1442,7 @@ def test_ordering(self):
             hit_sort_values = {}
             for order, response in responses.items():
                 response.raise_for_status()
-                hit_sort_values[order] = [accessor(hit) for hit in response.json()['hits']]
+                hit_sort_values[order] = [accessor(hit) for hit in response.json()['hits'] if accessor(hit) is not None]
```

Comment:
Now that the contents of `hit_sort_values` is filtered (`if accessor(hit) is not None`), adding verification that the lists aren't empty would make the sorting verification stronger.
```
            self.assertGreater(len(hit_sort_values['asc']), 0)
            self.assertGreater(len(hit_sort_values['desc']), 0)
```